tests/ast/data/pd-iloc.test

```diff
@@ -0,0 +1,17 @@
+## TEST CASE
+
+import modin.pandas as pd
+import snowflake.snowpark.modin.plugin
+from snowflake.snowpark import Session
+
+session = Session.builder.create()
+df = pd.DataFrame(
+    [[1, 2, 3, 4, 5, 6, 7], [8, 9, 10, 11, 12, 13, 14], [15, 16, 17, 18, 19, 20, 21]],
+    columns=["A", "B", "C", "D", "E", "F", "G"],
+)
+result = df.iloc[2, 2]
+print(result)
+
+## EXPECTED OUTPUT
+
+17
```

Comment:
Does this work? The test driver here runs the unparser, so we should be seeing something that looks like the original source.